instrumentation/powerjob-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/powerjob/v4_0/PowerJobHelper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.powerjob.v4_0;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import java.util.function.Predicate;
+import tech.powerjob.worker.core.processor.ProcessResult;
+
+public final class PowerJobHelper {
```

Comment:
I'd just inline the `startSpan` and `endSpan` methods. The instrumentations that use a similar helper class are usually more complicated.

Reply:
Thanks for your suggestion, I will streamline the code.